GitUI/UserControls/RevisionGrid/RevisionGridControl.cs

```diff
@@ -2425,7 +2429,40 @@ private void goToParentToolStripMenuItem_Click()
             }
         }
 
-        private void goToMergeBaseCommitToolStripMenuItem_Click(object sender, EventArgs e)
+        private void SelectNextForkPointAsDiffBase()
+        {
+            IReadOnlyList<GitRevision> revisions = GetSelectedRevisions();
+            if (revisions.Count == 0)
+            {
+                return;
+            }
+
+            GitRevision revision = revisions[revisions.Count - 1];
+            while (revision.IsArtificial)
+            {
+                revision = GetRevision(revision.FirstParentId);
+            }
+
+            do
+            {
+                GitRevision prevRev = GetRevision(revision.FirstParentId);
+                if (prevRev == null)
+                {
+                    break;
+                }
+
+                revision = prevRev;
+            }
+            while (!revision.Refs.Any(r => r.IsHead || r.IsRemote) && GetRevisionChildren(revision.ObjectId).Count == 1);
```

Comment:
Is it intentional that the following is selected?

![image](https://user-images.githubusercontent.com/6248932/155905073-f1618703-c66e-4410-ad8f-4b4c5a863b70.png)

Reply:
Yes - in a certain way - it is forked from that branch.
Just press the hotkey again until "master" appears in the FileStatusList:

![image](https://user-images.githubusercontent.com/36601201/156045952-8ea72e81-7eee-44d7-be6a-8216492ec605.png)

(I have not taken the time yet to describe the feature in more detail. Perhaps, it should be added to a menu, too.)

Reply:
OK
Could be added to the doc repo, maybe hard to find there (I did start preparing for 4.0).
Maybe menu too

Reply:
> Could be added to the doc repo, maybe hard to find there (I did start preparing for 4.0).

In order to avoid updating already updated docs, this PR can wait after 4.0. (There no risk for difficult merge conflicts.)

Reply:
Please push changes to my branch if you have any.